doc/faq.rst

```diff
@@ -88,13 +88,20 @@ The mouse is jumpy, slow, laggy when using SPICE
 Please be sure to install the SPICE guest tools from
 https://www.spice-space.org/download.html#windows-binaries.
 
-.. _the_mouse_doesnt_stay_aligned_with_the_host.:
+.. _mouse_desync_when_exiting:
 
-The mouse doesn't stay aligned with the host.
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The mouse doesn't stay aligned with the host when exiting the window
```

Comment:
Shouldn't this be when entering the window? Since that's when we try to move the guest cursor to an absolute position.

Reply:
Umm, yes... sorry, I missed this before I merged.

Reply:
Sorry, I misunderstood the situation